asv.conf.json

```diff
@@ -13,10 +13,10 @@
     "results_dir": "asv_results/results",
     "env_dir": "asv_env",
     "install_command": [
-        "in-dir={env_dir} python -mpip install {wheel_file}[map]"
+        "in-dir={env_dir} python -mpip install {wheel_file}[all]"
     ],
     "build_command": [
-        "python -mpip install extension_helpers numpy",
+        "python -mpip install -U setuptools wheel extension_helpers numpy",
```

Comment:
I am hoping this might fix the asv job.

Reply:
I was unable to replicate the build issue but since it failed on CITATION, I assume it might somehow be setup tools.